packages/addons/tools/network-tools/changelog.txt

```diff
@@ -2,6 +2,7 @@
 - bwm-ng: update to 0.6.3
 - tcpdump: update to 4.99.0
 - libpcap: revert to building with configure (107)
+- iftop: update to 2018-10-03 and fix compile with libpcap-1.10.0
```

Comment:
```suggestion
- iftop: update to 2018-10-03
```

and pls sort changelog abc for that entry

Reply:
Done. Fixed